vault/changelog.release.2022-01-04.md

```diff
@@ -0,0 +1,109 @@
+---
+id: MQpi87qLnzKK5iK3FlxUp
+title: 0.76
+desc: ''
+updated: 1641321329159
+created: 1641314638791
+---
+
+Dendron 0.76 has sprouted  🌱
+
+We're adding some super powers to your templates with [[template variables|dendron.topic.templates#template-variables]], starting with dates. Do you find yourself updating date values in the content of your inserted templates? Now that can be taken care of automatically:
+
+- Example: `Today is {{ CURRENT_YEAR }}.{{ CURRENT_MONTH }}.{{ CURRENT_DAY }}`
+- Template output: `Today is 2022.01.04`
+
+Have you found yourself switching between the `Dendron: Show Preview` and `Markdown: Open Preview` commands, in order to view Markdown files that aren't Dendron notes? Say no more: now `Dendron: Show Preview` can show you what your project `README.md` and other files look like, too.
```

Comment:
People haven't really been using markdown preview with dendron. would rephrase to add this as a capability instead. also a transition from previous paragraph

```
Another major quality of life improvement we're launching today is the ability to preview regular markdown using the Dendron Preview.
Now you can use the same Dendron preview on all your markdown files, even if they're not inside a Dendron workspace!
```